samples/minimum_criteria.rst

```diff
@@ -0,0 +1,98 @@
+.. _definition_and_minimum_criteria:
+
+Sample Definition and Minimum Criteria
+######################################
+
+Definition
+==========
+
+A sample is a basic Zephyr application that provides a brief overview of a feature, subsystem,
+or module. For example, Kernel services, drivers, protocols, etc.
+Samples should be limited in scope and should focus only on demonstrating non-trivial or
+essential aspects of the subsystem(s) or module(s) being highlighted.
+
+Minimum Sample Criteria
```

Comment:
Maybe all it "Sample Criteria" since it is setting both "lower" and "upper" criteria

Reply:
Done.